src/esp/sim/RenderInstanceHelper.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Meta Platforms, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "RenderInstanceHelper.h"
+#include "Simulator.h"
+#include "esp/assets/RenderAssetInstanceCreationInfo.h"
+#include "esp/core/Esp.h"
+
+#include <cassert>
+
+namespace esp {
+namespace sim {
+
+RenderInstanceHelper::RenderInstanceHelper(Simulator& sim,
+                                           bool useXYZWOrientations) {
+  sim_ = &sim;
+  isXYZW_ = useXYZWOrientations;
+}
+
+int RenderInstanceHelper::AddInstance(const std::string& assetFilepath,
+                                      int semanticId) {
+  esp::assets::AssetInfo assetInfo;
+  assetInfo.filepath = assetFilepath;
```

Comment:
Are we resolving the path downstream? Just making sure that `a/b/c` is equivalent to `a/b/../b/c`.

Reply:
I don't know, but this kind of path fixup makes more sense to me to do either (a) where the path is constructed or (b) where it is used for OS operations. Doesn't make sense to attempt the fixup here.